internal/app/machined/pkg/system/services/osd.go

```diff
@@ -88,7 +88,6 @@ func (o *OSD) Runner(r runtime.Runtime) (runner.Runner, error) {
 		{Type: "bind", Destination: "/tmp", Source: "/tmp", Options: []string{"rbind", "rshared", "rw"}},
 		{Type: "bind", Destination: constants.ConfigPath, Source: constants.ConfigPath, Options: []string{"rbind", "ro"}},
 		{Type: "bind", Destination: path.Dir(constants.ContainerdAddress), Source: path.Dir(constants.ContainerdAddress), Options: []string{"bind", "ro"}},
-		{Type: "bind", Destination: constants.DefaultLogPath, Source: constants.DefaultLogPath, Options: []string{"bind", "ro"}},
```

Comment:
I believe we don't need it as logs are served by machined

Reply:
:+1: